run_test.py

```diff
@@ -62,7 +62,7 @@
         conf = {
             'base': 'ubuntu14_py2',
             'cuda': 'cuda65',
-            'cudnn': 'cudnn2',
+            'cudnn': 'cudnn5',
```

Comment:
I want to test all versions of cudnn. cudnn2 is tested in py2 option. Instead cudnn5 in this test.